packages/samples/react/package.json

```diff
@@ -29,7 +29,7 @@
 		"@public-ui/themes": "1.7.0-rc.6",
 		"@types/react": "18.2.21",
 		"@types/react-dom": "18.2.7",
-		"@unocss/preset-mini": "0.55.7",
+		"@unocss/preset-uno": "^0.55.7",
```

Comment:
Bitte in KoliBri immer feste Versionen setzten. `"0.55.7"`

Reply:
Ich füge das zusätzlich mal in der root `.npmrc` hinzu, dann muss man nicht immer darauf achten (`save-exact=true`).